system/mp/mpcomm.cpp

```diff
@@ -2312,12 +2320,14 @@ CMPChannel *CMPServer::lookup(const SocketEndpoint &endpoint)
 }
 
 
-CMPServer::CMPServer(unsigned __int64 _role, unsigned _port)
+CMPServer::CMPServer(unsigned __int64 _role, unsigned _port, bool _listen)
 {
     RTsalt=0xff;
     role = _role;
     port = 0;   // connectthread tells me what port it actually connected on
     checkclosed = false;
+    listen = _listen;
```

Comment:
I think it would be cleaner if listen was a member of CMPConnectThread, avoiding the parent->listen + need to pass in to startPort etc.

Reply:
Yes, I agree.  Changed.